1.wave2_data/1.cellprofiler_ic_processing/create_loaddata_wave2_csv.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Create LoadData csv for CellProfiler `illum.cppipe` pipeline - Wave 2
+
+# ## Import libraries
+
+# In[1]:
+
+
+import os
+import pathlib
+
+
+# ## Helper function to run `pe2loaddata`
+
+# In[2]:
+
+
+def create_loaddata_csv(
```

Comment:
Is this the same function as what's found in `0.wave1_data/1.cellprofiler_ic_processing/create_loaddata_csv.py` ? If so, consider referencing this from a common location available to related implementation.

Reply:
It is! I can create the .py and use it in both.

Reply:
A utils section of this repo is made where i can create shared functions!